index.d.ts

```diff
@@ -13,6 +13,7 @@ declare const markdownItPandoc: (md: MarkdownIt, opts?: {
   subscript?:        boolean;
   superscript?:      boolean;
   task_lists?:       boolean;
+  highlightjs?:      boolean;
```

Comment:
Most (although not all) of these options are named after their pandoc counterparts. Although this one is named somewhat unfortunately: [`--no-highlight`](https://pandoc.org/MANUAL.html#option--no-highlight). But perhaps we should do instead then:

```suggestion
  highlight?:      boolean;
```